browser/src/control/Control.JSDialog.js

```diff
@@ -635,7 +635,7 @@ L.Control.JSDialog = L.Control.extend({
 		instance.startX = e.data.posx;
 		instance.startY = e.data.posy;
 		instance.updatePos = null;
-		instance.canHaveFocus = !instance.isSnackbar && instance.id !== 'busypopup' && !instance.isAutoCompletePopup;
+		instance.canHaveFocus = !instance.isSnackbar && instance.id !== 'busypopup' && !instance.isAutoCompletePopup && instance.id !== 'autoFillPreviewTooltip';
```

Comment:
we want to avoid hardcoding the ids to the engine, instead please put `canHaveFocus` into the JSON :)

Reply:
added `canHaveFocus: false,` but it seems we don't need it anymore. 
`instance.noOverlay` seems enough for unblocking the autofill.